### ISSUE TITLE
High disk I/O

### ISSUE THREAD

**Issue by sclevine** [vmware-archive/pcfdev#195]
We've received a report that the PCF Dev VM eats up a lot of disk I/O while idle. This is a placeholder issue to investigate.

**Comment by sclevine** [comment]
We received a report that errors like this can be due to poor I/O performance on some systems that can be fixed by reconfiguring the Virtualbox VM like so:
```
vboxmanage storagectl pcfdev-v0.447.0 --name SATA --hostiocache on
```
We'll investigate configuring the VM like this in the next release.

**Comment by mdelillo** [comment]
This has been fixed in the latest release (v0.26.0).
